README.md

```diff
@@ -290,6 +290,12 @@ Node has a [schedule](https://github.com/nodejs/Release#release-schedule) for lo
 
 Any time your local copy of `nvm` connects to https://nodejs.org, it will re-create the appropriate local aliases for all available LTS lines. These aliases (stored under `$NVM_DIR/alias/lts`), are managed by `nvm`, and you should not modify, remove, or create these files - expect your changes to be undone, and expect meddling with these files to cause bugs that will likely not be supported.
 
+To get the latest LTS version of node and migrate your existing installed packages, use
+
+```sh
+nvm install lts/* --reinstall-packages-from=current
```

Comment:
```suggestion
nvm install --lts --reinstall-packages-from=current
```
this is also correct; it's probably a better choice so the `*` doesn't accidentally get expanded by a shell?

Reply:
Sure. Looks good

Reply:
you marked this as resolved, but didn't update the text. i went ahead and took care of it for you.